src/Azs.ContainerRegistry.Admin/examples/Read-AzsContainerRegistry.md

```diff
@@ -0,0 +1,19 @@
+### Example 1: Read List Azs Container Registries
+```powershell
+PS C:\> Read-AzsContainerRegistry
```

Comment:
Why are we calling this with Read instead of Get?

Reply:
Fixed